br/tests/lightning_csv/run.sh

```diff
@@ -41,3 +41,9 @@ for BACKEND in tidb local; do
   check_not_contains 'id:'
 
 done
+
+run_lightning --backend local -d "tests/$TEST_NAME/errData" --log-file "$TEST_DIR/lightning-err.log"
+# err content presented
+grep ",7,8" "$TEST_DIR/lightning-err.log"
+# pos should not set to end
+grep "[\"syntax error\"] [pos=30]" "$TEST_DIR/lightning-err.log"
```

Comment:
how to use the value 30? seems the quote is at offset 22

Reply:
Same as above, I changed the test file and forgot to change test script😭

Reply:
5fd6c58ac